.prettierrc.json

```diff
@@ -1,5 +1,4 @@
 {
-    "trailingComma": "es6",
```

Comment:
My bad, sorry. Replace with es5 or check docs for latest version that matches with the tsconfig

Reply:
es5 is the latest version

Reply:
We can remove it altogether, it's enabled by default